src/pages/resultsView/oncoprint/AddClinicalTracks.tsx

```diff
@@ -210,30 +130,19 @@ export default class AddClinicalTracks extends React.Component<
                         id={Tab.CLINICAL}
                         linkText={Tab.CLINICAL}
                         hide={
-                            !this.options.isComplete ||
-                            this.options.result!.clinical.length === 0
+                            !this.props.options.isComplete ||
+                            this.props.options.result!.clinical.length === 0
                         }
                     >
                         {this.addClinicalTracksMenu.component}
                     </MSKTab>
                     <MSKTab
```

Comment:
So, this is what I was looking for.  There is still a possibility that there will be more than one tab because of the custom charts tab.  Thus, we need to keep the menu.   I see that we are already hiding the menu when there is only one option, so it seems like no change is necessary here.    (If there was only ever a single tab, it would make sense to just get rid of the tab structure altogether).

Reply:
@gblaih @inodb now that I think about it, I don't think the custom charts belongs in the clinical tab submenu either.  so, really we should have 4 top level tabs and NO subtabs ever.